chart/templates/deployment.yaml

```diff
@@ -48,6 +48,10 @@ spec:
               value: {{ .Values.log_level | quote }}
             - name: OBS_RUNNER_LOG_FORMAT
               value: {{ .Values.log_format | quote }}
+            {{- if .Values.default_monitor_job_timeout }}
```

Comment:
fwiw for future reference it would be great to split helm changes vs code changes into seperate commits